<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermalBox</class>
 <widget class="QWidget" name="ThermalBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thermal Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../PythonGecko/thermalResistor.jpg</normaloff>../PythonGecko/thermalResistor.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="thermDoneBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thermCnclBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>Inputs        :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="displayLabel">
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>number</string>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>R (J-C)       :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="rjc">
        <property name="lineWidth">
         <number>-1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="tabStopWidth">
         <number>80</number>
        </property>
        <property name="cursorWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> R (C-S)      :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="rcs"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Cth             :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="cth"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>thermCnclBtn</sender>
   <signal>clicked()</signal>
   <receiver>thermCnclBtn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
